tardis-setups/2014/plot_tardis_example.py

```diff
@@ -0,0 +1,53 @@
+"""
+Tardis 2014 Example
+==========================
+
+This is the example output
+"""
+
+
+from tardis import run_tardis
+from tardis.io.config_reader import Configuration
+from tardis.io.atom_data.util import download_atom_data
+import matplotlib.pyplot as plt
+
+import sys
+
+sys.path.append("../")
```

Comment:
We needed a couple of functions to be used across all the papers. So we needed to put them in a separate folder so that all the files could import them. The only way to access those functions is through this function.